sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -271,14 +267,16 @@ export class ApplicationService extends RecordDataModelService<Application> {
       newApplication.creator = auditUser;
       newApplication.studentAssessments = [originalAssessment];
       newApplication.currentAssessment = originalAssessment;
+
+      // Updates the current assessment status to cancellation requested.
+      application.currentAssessment.studentAssessmentStatus =
+        StudentAssessmentStatus.CancellationRequested;
+      application.currentAssessment.modifier = { id: auditUserId } as User;
```

Comment:
There is already an `auditUser` variable that can be used here instead `{ id: auditUserId } as User`.